stonesoup/models/clutter/clutter.py

```diff
@@ -42,6 +42,9 @@ class ClutterModel(Model, ABC):
         "The default defines the space for a uniform distribution in 2D. The call "
         "`np.array([self.distribution(*arg) for arg in self.dist_params])` "
         "must return a numpy array of length equal to the number of dimensions.")
+    random_state: Optional[np.random.RandomState] = Property(
+        default=np.random.RandomState(1),
```

Comment:
The issue with having this as a default, is run to run you'll get the same results, and I think it'd be better that by default the global random state is used. Also, this default will be shared by all instances, which may have unexpected issues.

Something like in Gaussian Model may be better:
https://github.com/dstl/Stone-Soup/blob/2de2fc0a7075c9898dffb841658076cfbe781a7e/stonesoup/models/base.py#L211-L215

Reply:
That's a very good point, I hadn't thought of that. I've made this change like you suggested. However, I did keep the seed as either an integer seed or a numpy random state (or default None). I thought this way the user can use the same random state for the poisson random variable and the distribution attribute. For example:
```
rs = numpy.random.RandomState(seed=1)  
clutter_model = ClutterModel(  
     distribution=rs.uniform,  
     seed=rs,  
     ...  
)
```

Does that seem ok?

Reply:
Sorry, I don't quite understand why this change made the tests fail. It seems to be failing Test 39, installing dependencies.

Reply:
> Sorry, I don't quite understand why this change made the tests fail. It seems to be failing Test 39, installing dependencies.

CircleCI seems to be having issues lately with the dependency cache getting corrupted. Seems I can't reset the cache on PRs from forks. Will have to look into a solution, but just ignore failures for now.

Reply:
ok, thank you!